src/index.js

```diff
@@ -1,28 +1,142 @@
+/* eslint-disable no-use-before-define */
```

Comment:
- [ OPTIONAL] My kind suggestion is that you should always avoid disabling linters. Linters ensure that we write good & uniform quality code. 😄